installer/terraform/chefProvisioner.tf

```diff
@@ -112,14 +130,15 @@ resource "null_resource" "configureExistingJenkinsServer" {
   }
 }
 
-data "external" "gitlabcontainer" {
+data "external" "gitlabconfig" {
   count = "${var.scmgitlab}"
-  program = ["bash", "${var.configureGitlab_cmd}"]
+  program = ["python3", "${var.configureGitlab_cmd}"]
 
   query = {
     passwd = "${var.cognito_pool_password}"
-    ip = "${aws_lb.alb_ecs_gitlab.dns_name}"
-    gitlab_admin = "${lookup(var.scmmap, "scm_username")}"
```

Comment:
By default the gitlab username is root, so we have to update the username https://github.com/tmobile/jazz-installer/blob/9e7e14f0467e40537ef8f9892c7439bec349f7a7/installscripts/jazz-terraform-unix-noinstances/scripts/configure-gitlab.sh#L9

Also, the username should be  replaced with '[^a-zA-Z0-9_-]', '-' 
- https://github.com/tmobile/jazz-installer/blob/9e7e14f0467e40537ef8f9892c7439bec349f7a7/installscripts/wizard/scenarios/support/jazz_gitlab.py#L26

Reply:
What's the point of changing the name of the default `root` user? I'm assuming "security", but that's unlikely to make a real difference to anyone running a dictionary attack or whatever against the login form, as long as we use a nondefault password.

Reply:
@bleggett Earlier what we did is, update the root user to a specific username and used that for login. Now Gitlab login is not working using the email and password. By default there wont be any email in gitlab and username would be root.  In the final stack output it is the email that is displaying to login. Also in the jenkins credentials, username is email. So all the builds will fail.

Reply:
I thought Jenkins was using a personal access token, not username/password?

Reply:
We should revisit this, because it feels like the way it's currently being done is wrong, but for the time being I have made it work like it does currently.